doc/source/data/working-with-images.rst

```diff
@@ -0,0 +1,274 @@
+.. _working_with_images:
+
+Working with Images
+===================
+
+With Ray Data, you can easily read and transform large image datasets.
+
+This guide shows you how to:
+
+* :ref:`Read images <reading_images>`
+* :ref:`Transform images <transforming_images>`
+* :ref:`Perform inference on images <performing_inference_on_images>`
+* :ref:`Save images <saving_images>`
+
+.. _reading_images:
+
+Reading images
+--------------
+
+Ray Data can read images from a variety of formats.
+
+To view the full list of supported file formats, see the
+:ref:`Input/Output reference <input-output>`.
+
+.. tab-set::
+
+    .. tab-item:: Raw images
+
+        To load raw images like JPEG files, call :func:`~ray.data.read_images`.
+
+        .. note::
+
+            :func:`~ray.data.read_images` uses
+            `Pillow <https://pillow.readthedocs.io/en/stable/index.html>`_. For a list of
```

Comment:
I would call this PIL. I think this is what most people refer to it as, for example in the pytorch docs.